### ISSUE TITLE
wAnchor config and wLogits config

### ISSUE THREAD

**Issue by z00kiii** [dimitymiller/openset_detection#6]
Hi, I'm trying to implement your work with a coco dataset in my own project and I'm a little bit confused about the ...wAnchor and the ...wLogits configs. When looking at your README and code you train a model with the ...wAnchor config. However for evalutaion you use the ...wLogits config. 
I guess you train weights with the Anchor Loss (...wAnchor config) and for inference you use the trained weights but with the ...wLogits config without Anchor Loss.
As I'm not 100% sure about it by reading the paper I wanted to ask you directly.

**Comment by dimitymiller** [comment]
Hi z00kii,

You're correct -- the wAnchor config is for during training, and uses the Anchor loss. 

When testing the model, the normal bbox_head 'Shared2FCBBoxHead' does not return the logits with a prediction. We need these when testing in order to model the GMMs. The wLogits config was created for this reason to be used when testing, and uses an altered 'Shared2FCBBoxHeadLogits' bbox_head that returns the logits alongside the detections. 

Hope that makes sense! It is not a change on the method, and is an implementation detail.
